collections.js

```diff
@@ -118,8 +118,36 @@ LinkedList.prototype.find = function(value) {
     node = node.next;
   }
   return null;
+};
+
+//------------------------------------------------------------------------------
+// Simple Queue.
+
+function SimpleQueue() {
```

Comment:
Darn, this isn't my latest code.  In my latest, I just use Queue.  I'll update to send you my latest... hopefully soon!